package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wowza-webrtc-player",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
Best practice is to update the version once you want to publish the package.
It will avoid problem doing parallels PRs.